src/main/kotlin/com/exactpro/th2/common/message/MessageUtils.kt

```diff
@@ -457,14 +492,16 @@ fun getDebugString(className: String, ids: List<MessageID>): String {
 }
 
 @JvmOverloads
-fun com.google.protobuf.MessageOrBuilder.toJson(short: Boolean = true): String = JsonFormat.printer().includingDefaultValueFields().let {
-    (if (short) it.omittingInsignificantWhitespace() else it).print(this)
-}
+fun com.google.protobuf.MessageOrBuilder.toJson(short: Boolean = true): String =
+    JsonFormat.printer().includingDefaultValueFields().let {
+        (if (short) it.omittingInsignificantWhitespace() else it).print(this)
+    }
 
-fun <T: com.google.protobuf.Message.Builder> T.fromJson(json: String) : T = apply {
+fun <T : com.google.protobuf.Message.Builder> T.fromJson(json: String): T = apply {
     JsonFormat.parser().ignoringUnknownFields().merge(json, this)
 }
 
+@Deprecated("Moved to common-utils-j/com.exactpro.th2.common.utils.message.MessageUtils")
```

Comment:
It is kotlin, so we can add a replacement suggestion to the deprecation annotation. It does not check the presents of the class but will it automates the migration